components/automate-deployment/pkg/bootstrap/deployment_service_cmd.go

```diff
@@ -47,7 +47,11 @@ func (ds *DeploymentServiceCommand) CanBootstrap() bool {
 	return ds.HasCapability("bootstrap")
 }
 
-func (ds *DeploymentServiceCommand) DeployService(ctx context.Context, config *dc.ConfigRequest, m manifest.ReleaseManifest) error {
+func (ds *DeploymentServiceCommand) CanUnpackBootstrapBundle() bool {
+	return ds.HasCapability("bootstrap")
```

Comment:
I think we should give this a new capability name, maybe "bootstrap-with-bundle"

Reply:
oops